Create artifact package (#38)
aragon/admin-plugin: packages/artifacts/README.md

# Admin Plugin artifacts

This package contains the ABIs of the OSx Admin plugin, as well as the address of the plugin repository on each network. 

## Installation

```sh
yarn add @aragon/admin-plugin-artifacts
```

## Documentation

> You can find all documentation regarding how to use this plugin in [Aragon's documentation here](https://docs.aragon.org/).

Reviewer: ```suggestion
You can find all documentation regarding how to use this plugin in [Aragon's documentation here](https://docs.aragon.org/admin/1.x/index.html).
```

> # Artifacts Package for Admin Plugin

Reviewer 2: ```suggestion
# Admin Plugin artifacts
```

Author: amended

> This package is responsible for generating and storing ABIs for smart contracts. It checks out the specified branch, compiles the contracts, and saves the ABIs in the `src/abis/` directory. The package can then be published to NPM for use in other projects.

Reviewer 2: ```suggestion
This package contains the ABIs of the OSx Admin plugin, as well as the address of the plugin repository on each network. Install it with:

```sh
yarn add @aragon/admin-plugin
```

Reviewer 2: GitHub's diff is omitting the closing ticks ```
These would need to be added manually

Author: amended

> ## Installation
> 
> Run the following command to install dependencies:
> ```sh
> yarn install
> ```
> 
> ## Usage
> ###  Generate ABIs
> Run:
> ```sh
> yarn generate
> ```
> This will:
> 1. Check out the `packages/contracts`.
> 2. Install dependencies.
> 3. Compile contracts using Hardhat.
> 4. Generate ABIs using Wagmi.
> 5. Save the ABIs in `src/abis/`.

Reviewer 2: ```suggestion
## Development

Install the dependencies by running `yarn install`

###  Generate the ABIs

Generate the ABI artifacts by running `yarn generate`

This will:
1. Check out the `packages/contracts`.
2. Install dependencies.
3. Compile contracts using Hardhat.
4. Generate ABIs using Wagmi.
5. Save the ABIs in `src/abis/`.
```

Author: became irrelevant
